framework/src/wifi_manager/wifi_manager.c

```diff
@@ -974,3 +982,40 @@ wifi_manager_result_e wifi_manager_scan_ap(void)
 	wifi_manager_result_e res = _handle_request(&msg);
 	return res;
 }
+
+
+wifi_manager_result_e wifi_manager_save_config(wifi_manager_ap_config_s *config)
+{
+	LOCK_WIFIMGR;
```

Comment:
This mutex is required to protect wifi_manager operations and related common variables.
The file, where the config is saved, can be protected by another mutex.

Reply:
Ok, I will fix. I think that these functions doesn't require mutex. because it doesn't refer global variables